docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.7"
+
+services:
+  mongo:
+    image: mongo:4.2.2-bionic
+    volumes:
+      - data_mongo:/data/db
```

Comment:
As this is just used for testing purposes, I guess there is no need to retain test data in a Docker volume.

Reply:
Fixed in a1d553443c53a3f75748c741b4a04d0869ec9f61